blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -12,37 +12,40 @@ import * as pushEventUtil from './util/push-event-util';
 const { object, array, func, node, string } = PropTypes;
 
 class OrganizationPipelines extends Component {
-
-    // FIXME: IMO the following should be dropped
+    constructor(props) {
+        super(props);
+        this.state = {};
```

Comment:
and? this.setState will not care.

Anyway I just looked for the usage of this.state and you only use it for one compare, but IMO you could do without, only comparing the orgaName for current props and nextProps.

No need for state at all.